File: Lista 05/exercício 09.py
#Criar um algoritmo que receba 10 números positivos e escreva a raiz quadrada de cada número. Para cada entrada de dados deverá haver um trecho de proteção para que um número negativo não seja aceito.

vetor = []

for i in range(10):
    n = int(input("Digite um número: "))
    vetor.append(n)
for i in range(10):
    if vetor[i] < 0:
        vetor[i] = int(input(f"Digite um novo valor pois {vetor[i]} é inválido "))

for i in range(10):
    raiz = vetor[i] ** (1/2)
    print(raiz)

File: Lista 13/exercício 09.c
// Crie um algoritmo que receba um número inteiro. Escreva uma função que retorne se este valor é positivo ou negativo.

#include <stdio.h>
#include <stdlib.h>

int positivonegativo(int n)
{
    if (n > 0)
    {
        return (1);
    }
    if (n < 0)
    {
        return (0);
    }
}

int main()
{
    int n;

    printf("Digite um número: ");
    scanf("%d", &n);

    if (n == 0)
    {
        printf("é um número neutro\n");
    }
    else if (positivonegativo(n) == 1)
    {
        printf("\nNúmero positivo\n");
    }
    else
    {
        printf("\nNúmero negativo\n");
    }

    system("PAUSE");
    return 0;
}